qualtran/bloqs/data_loading/one_hot_encoding.py

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import attrs
+import cirq
+from attr import field
+from numpy._typing import NDArray
+
+from qualtran import GateWithRegisters, QAny, QUInt, Signature
+from qualtran.bloqs.basic_gates import TwoBitCSwap
+
+
+@attrs.frozen
+class OneHotEncoding(GateWithRegisters):
+    """
```

Comment:
One line description here, directly after the """ (no new line)

Reply:
Are you saying a new line in between """ and the description? I don't currently see one.

Reply:
the text and the triple-quotes should be on the same line